src/views/ModelPlan/Documents/AddDocument/LinkDocument.tsx

```diff
@@ -0,0 +1,399 @@
+import React, { useContext, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory, useParams } from 'react-router-dom';
+import { useMutation } from '@apollo/client';
+import { Button, Fieldset, Label, TextInput } from '@trussworks/react-uswds';
+import { Field, Form, Formik, FormikProps } from 'formik';
+
+import BooleanRadio from 'components/BooleanRadioForm';
+import Alert from 'components/shared/Alert';
+import { ErrorAlert, ErrorAlertMessage } from 'components/shared/ErrorAlert';
+import FieldErrorMsg from 'components/shared/FieldErrorMsg';
+import FieldGroup from 'components/shared/FieldGroup';
+import { RadioField } from 'components/shared/RadioField';
+import RequiredAsterisk from 'components/shared/RequiredAsterisk';
+import TextAreaField from 'components/shared/TextAreaField';
+import TextField from 'components/shared/TextField';
+import useMessage from 'hooks/useMessage';
+import LinkNewPlanDocument from 'queries/Documents/LinkNewPlanDocument';
+import { LinkNewPlanDocument as LinkNewPlanDocumentType } from 'queries/Documents/types/LinkNewPlanDocument';
```

Comment:
Sorry I didn't catch this before but moving forward we want to use the new codegen.  IIRC Clay mentioned that the old package is no longer being maintained and we should write all new queries with the new toolset